web/src/ui_init.js

```diff
@@ -221,7 +222,7 @@ export function initialize_kitchen_sink_stuff() {
         if (delta < 0 && message_viewport.at_rendered_top()) {
             navigate.up();
         } else if (delta > 0 && message_viewport.at_rendered_bottom()) {
-            navigate.down();
+            navigate.down(false, true);
```

Comment:
It might be nice to pass these two options by name rather than positional values.